src/main/java/org/folio/service/consortia/CrossTenantUserServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.service.consortia;
+
+import io.vertx.core.Future;
+import org.folio.rest.jaxrs.model.User;
+
+import java.util.Map;
+
+public interface CrossTenantUserServiceImpl {
+
+  /**
+   * Searching user in CrossTenant if it isn't CrossTenant will return null
```

Comment:
Suggest to rename to: Firstly search user id and tenant id using /user-tenant endpoint  from mod-users central tenant by fields like: username, email, phone number. If found - use tenantId and userId to find user in the tenant where user was created